packages/tame-metering/CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+### [2.0.13](https://github.com/Agoric/agoric-sdk/compare/@agoric/tame-metering@2.0.12...@agoric/tame-metering@2.0.13) (2021-09-23)
```

Comment:
we don't need tame-metering any more, do we?

Reply:
Right.